test/unit/keystore_test.exs

```diff
@@ -92,4 +92,34 @@ defmodule Unit.KeystoreTest do
     {:ok, signature} = Bls.sign(privkey, digest)
     assert Bls.valid?(pubkey, digest, signature)
   end
+
+  test "eip scrypt without pubkey test vector" do
+    decoded_json = Jason.decode!(@scrypt_json)
+    Map.delete(decoded_json, "pubkey")
```

Comment:
`Map.delete` returns the modified map. The json with the deleted pubkey is being discarded here.

Reply:
Done